test/AddCommandTest.py

```diff
@@ -200,7 +200,7 @@ def test_add_dep9(self):
         command = ListCommand.ListCommand(["Bar"], self.todolist, self.out, self.error)
         command.execute()
 
-        self.assertEqual(self.output, "|kbn| {today} Bar p:1 +Project\n|kbn| {today} Bar +Project\n".format(today=self.today))
+        self.assertEqual(self.output, "|kbn| {today} Bar p:1 +Project\n|kbn|   {today} Bar +Project\n".format(today=self.today))
```

Comment:
I suspect it's related to [this commit](https://github.com/MinchinWeb/topydo/commit/4bc70e336aaeb14ced1ce7df3c5f10421da3e537)? If so, it would be best to logically combine commits: if you break a test case with a code change, add the fixed test in the same commit.